bundles/org.openhab.voice.voskstt/README.md

```diff
@@ -3,7 +3,7 @@
 Vosk STT Service uses [vosk-api](https://github.com/alphacep/vosk-api) to perform offline speech-to-text in openHAB.
 
 [Vosk](https://alphacephei.com/vosk/) is an offline open source speech recognition toolkit.
-It enables speech recognition for 20+ languages and dialects - English, Indian English, German, French, Spanish, Portuguese, Chinese, Russian, Turkish, Vietnamese, Italian, Dutch, Catalan, Arabic, Greek, Farsi, Filipino, Ukrainian, Kazakh, Swedish, Japanese, Esperanto.
+It enables speech recognition for 20+ languages and dialects - English, Indian English, German, French, Spanish, Portuguese, Chinese, Russian, Turkish, Vietnamese, Italian, Dutch, Catalan, Arabic, Greek, Farsi, Filipino, Ukrainian, Kazakh, Swedish, Japanese, Esperanto. For more information about Vosk, see the [Vosk speech recognition toolkit website](https://alphacephei.com/vosk/).
```

Comment:
```suggestion
It enables speech recognition for 20+ languages and dialects - English, Indian English, German, French, Spanish, Portuguese, Chinese, Russian, Turkish, Vietnamese, Italian, Dutch, Catalan, Arabic, Greek, Farsi, Filipino, Ukrainian, Kazakh, Swedish, Japanese, Esperanto.
For more information about Vosk, see the [Vosk speech recognition toolkit website](https://alphacephei.com/vosk/).
```

This is the same link as in line 5. Is that line sufficient or should the two sentences be merged somehow to not repeat?

Reply:
I think it is fine to leave it. When reading it is not anoying or and the link is duplicate, the text is.